extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -426,11 +424,16 @@ export class BookTocManager implements IBookTocManager {
 				this.tableofContents.push(this.newSection);
 				await fs.writeFile(targetBook.tableOfContentsPath, yaml.safeDump(this.tableofContents, { lineWidth: Infinity, noRefs: true, skipInvalid: true }));
 			} else {
-				await this.updateTOC(targetBookVersion, targetBook.tableOfContentsPath, targetSection, this.newSection);
+				await this.updateTOC(targetBook.book.version, targetBook.tableOfContentsPath, targetSection, this.newSection);
 			}
 		}
 	}
 
+	public async removeNotebook(element: BookTreeItem): Promise<void> {
+		const findSection = { file: element.book.page?.file?.replace(/\\/g, '/'), title: element.book.page?.title };
```

Comment:
I seem to see a lot of this replacing of backslashes in the path - is this not something that would be better done once when you load the book and then reuse everywhere that needs it?